triples-generator/src/main/scala/io/renku/triplesgenerator/events/categories/triplesgenerated/projectinfo/ProjectEventsFinder.scala

```diff
@@ -0,0 +1,100 @@
+package io.renku.triplesgenerator.events.categories.triplesgenerated.projectinfo
```

Comment:
I think we need a preamble here. Maybe just run `Test / compile` on the whole project